### ISSUE TITLE
NL Design System Baseline: juni 2026

### ISSUE THREAD

**Issue by RenateRoke** [nl-design-system/documentatie#3844]
We hebben een nieuwe versie nodig van de NL Design System baseline:

De laatste is:

## Acceptatie-criteria

- [ ] De huidige versienummers van browsers en hulpmiddelen zijn gedocumenteerd in een nieuwe pagina, op basis van de regels beschreven bij [Baseline ➝ Software om mee te testen](https://nldesignsystem.nl/baseline/#software-om-mee-te-testen)
- [ ] De nieuwe editie van de baseline is aangekondigd in Slack.
- [ ] GitHub Issue voor volgende baseline editie is aangemaakt en ingepland in een volgende sprint

**Comment by RenateRoke** [comment]
[Chrome stable desktop 149](https://chromereleases.googleblog.com/2026/06/stable-channel-update-for-desktop_0153744567.html)
[NVDA 2026.1.1](https://www.nvaccess.org/post/nvda-2026-1-1/)
[Chrome 149 Android](https://chromereleases.googleblog.com/2026/06/chrome-for-android-update_01005552403.html)
[TalkBack 17](https://support.google.com/accessibility/android/answer/17069786?hl=en&ref_topic=10601875&sjid=13227526755757837604-EU)
[Firefox 151](https://www.firefox.com/en-US/firefox/151.0/releasenotes/)

**Comment by RenateRoke** [comment]
[Baseline juni gedeeld op nl-design-system op Slack](https://codefornl.slack.com/archives/C7KDQRXMZ/p1781262645198409)
